### DISCUSSION TITLE
Run this docker image as non root user

### DISCUSSION THREAD

**Discussion by katuka2** [ix-ai/smtp#33] [Q&A]
Looks like this image is running as root user.
How to run this docker image as non-root user? What changes needed in dockerfile?

**Comment by katuka2** [comment]
@tlex any inputs on this ?

**Reply by tlex** [reply]
I expect, you need to look into getting exim to start as non-root. This image is only a packaging of exim.

I haven't tested it, nor have time for it. But I'm happy to accept PRs for it
